src/products/components/ProductTile/ProductTile.tsx

```diff
@@ -15,66 +16,92 @@ const commonThumbnailProps = {
   borderColor: "neutralHighlight",
   borderStyle: "solid",
   borderWidth: 1,
-  marginBottom: 8,
+  marginBottom: 4,
   borderRadius: 3,
   aspectRatio: "1 / 1",
 } as const;
 
 export const ProductTile: React.FC<ProductTileProps> = ({
   product,
   onClick,
-}) => (
-  <Box
-    display="flex"
-    flexDirection="column"
-    onClick={onClick}
-    cursor="pointer"
-    backgroundColor={{ hover: "interactiveNeutralHighlightHovering" }}
-    borderRadius={4}
-    padding={5}
-    data-test-id={`product-tile-${product.id}`}
-  >
-    {product.thumbnail ? (
-      <Box
-        {...commonThumbnailProps}
-        as="img"
-        alt={product.name}
-        objectFit="scale-down"
-        src={product.thumbnail.url}
-      />
-    ) : (
-      <Box
-        {...commonThumbnailProps}
-        display="flex"
-        alignItems="center"
-        justifyContent="center"
-        backgroundColor="surfaceNeutralSubdued"
-      >
-        <Box __width="40%" __height="40%">
-          <ProductsIcons
-            size="fill"
-            color="iconNeutralDisabled"
-            data-test-id={`placeholder-svg-${product.id}`}
-          />
+}) => {
+  const { spacing } = useTheme();
```

Comment:
**suggestion**: if you migrate to `vars` from `@salero/macaw-ui/next` (`import { vars } from "@saleor/macaw-ui/next";` you shouldn't need `spacing` here